keps/sig-node/5304-dra-attributes-downward-api/kep.yaml

```diff
@@ -0,0 +1,47 @@
+title: DRA Device Attributes Downward API
+kep-number: 5304
+authors:
+  - "@alaypatel07"
+owning-sig: sig-node
+participating-sigs: []
+status: provisional
+creation-date: 2025-10-02
+reviewers:
+  - "@klueska"
+  - "@pohly"
+approvers:
+  - "@johnbelamaric"
```

Comment:
I am not a SIG-node approver

Reply:
swapped you to reviewers list, added @mortent to reviewers as well and added @klueska to approver.